lib/tune.cpp

```diff
@@ -229,10 +229,11 @@ namespace quda {
       strncpy(tmp, key.aux, 13);
       bool is_policy_kernel = strncmp(tmp, "policy_kernel", 13) == 0 ? true : false;
       bool is_policy = (strncmp(tmp, "policy", 6) == 0 && !is_policy_kernel) ? true : false;
+      bool is_nested_policy = (strncmp(tmp, "nested_policy", 6) == 0) ? true : false; // nested policies not included
```

Comment:
Where is nested_policy used?

Reply:
2-d multi reductions use nested policies: I've added a transpose tuner, which tests if it is better to do an `mxn` or `nxm` multireduction.  It's nested of course since below that there is also a policy tuner for the tile size.